app/controllers/nodes_controller.rb

```diff
@@ -114,6 +113,26 @@ def tree
   end
 
   private
+
+  def set_evidence_default_columns
+    default_field_names = ['Title', 'Created', 'Updated'].freeze
+    extra_field_names = ['Created by'].freeze
+
+    note_dynamic_fields = dynamic_field_names(@node.notes)
+    evidence_dynamic_fields = dynamic_field_names(@node.evidence)
+
+    rtp = current_project.report_template_properties
+    rtp_default_evidence_fields = rtp ? rtp.evidence_fields.default.field_names : []
+
+    @note_columns     = default_field_names | note_dynamic_fields | extra_field_names
+    @evidence_columns = default_field_names | rtp_default_evidence_fields | evidence_dynamic_fields | extra_field_names
+
+    @default_columns = {
+      evidence: rtp_default_evidence_fields.presence || default_field_names,
+      note: default_field_names
+    }
```

Comment:
I don't love that we keep separate `@note_columns` and `@evidence_columns` but merge them into a single `@default_columns`, I think`@default_note_columns` and `@default_evidence_columns` makes a bit more sense?

Reply:
Done